src/containers/MultidisciplinarySubject/MultidisciplinarySubjectArticle.jsx

```diff
@@ -21,25 +21,56 @@ import { useGraphQuery } from '../../util/runQueries';
 import { topicQueryWithPathTopics } from '../../queries';
 import { scrollToRef } from '../SubjectPage/subjectPageHelpers';
 import { getUrnIdsFromProps } from '../../routeHelpers';
+import { fetchGrepCodeTitle } from '../../util/grepApi';
 
 const filterCodes = {
-  'Folkehelse og livsmestring': 'publicHealth',
-  'Demokrati og medborgerskap': 'democracy',
-  'Bærekraftig utvikling': 'climate',
+  TT1: 'publicHealth',
+  TT2: 'democracy',
+  TT3: 'climate',
+};
+
+const subjectPaths = {
+  TT1:
+    '/subject:d1fe9d0a-a54d-49db-a4c2-fd5463a7c9e7/topic:3cdf9349-4593-498c-a899-9310133a4788/',
+  TT2:
+    '/subject:d1fe9d0a-a54d-49db-a4c2-fd5463a7c9e7/topic:077a5e01-6bb8-4c0b-b1d4-94b683d91803/',
+  TT3:
+    '/subject:d1fe9d0a-a54d-49db-a4c2-fd5463a7c9e7/topic:a2f5aaa0-ab52-49d5-aabf-e7ffeac47fa2/',
```

Comment:
@gunnarvelle Klarte ikke å finne en kobling mellom grepkodene og topics/subjects i taxonomy, er det noe? Hardkoda midlertidig inn pathen til emnene i prod, men det er jo ikke helt holdbart. 

Også litt dumt at ikke alle hovedemnene ligger i test for dette, kan jeg endre på det i strukturredigeringa?

Reply:
Vi har dessverre ikkje en slik kobling. Vi kunne gjort det dynamisk ved å sette kodene på emner i taksonomien også, men id til faget tverrfaglige tema er uansett hardkoda.

Vi får ikkje endra tverrfaglige tema via strukturredigering og få samme ider, men eg kan migrere ting fra staging til test og på den måten få det likt.

Reply:
Nei okei, hva tror du er beste løsning her da? Hvis vi setter på kodene i taksonomi så går det jo an å lage en liste med de ulike id-ene for tverrfaglig tema til de forskjellige miljøene og hente avhengig av hvilket miljø man er i? Blir hvertfall litt færre ulike id-er å forholde seg til. Eller er det ikke så farlig at det ikke funker perfekt i test? Ser ut som emnene har samme path i staging og prod hvertfall.

Reply:
Har migrert data så då er du ett steg videre å kunne teste dette i test.